src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CollectionCommand.java

```diff
@@ -20,5 +20,7 @@
   @JsonSubTypes.Type(value = InsertManyCommand.class),
   @JsonSubTypes.Type(value = UpdateManyCommand.class),
   @JsonSubTypes.Type(value = UpdateOneCommand.class),
+  @JsonSubTypes.Type(value = AddIndexCommand.class),
```

Comment:
nit: should these be added in alphabetic order like others are (except for `EstimatedDocumentCountCommand`)?